scripts/upload.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+#-*- coding:utf-8 -*-
+
+import sys
+import os
+from qiniu import Auth, put_file, etag, urlsafe_base64_encode
+import qiniu.config
+from qiniu.compat import is_py2, is_py3
+
+import logging
+import boto3
+from botocore.exceptions import ClientError
+
+QINIU_ACCESS_KEY = os.getenv('QINIU_ACCESS_KEY')
+QINIU_SECRET_KEY = os.getenv('QINIU_SECRET_KEY')
+QINIU_BUCKET_NAME = os.getenv('QINIU_BUCKET_NAME')
+AWS_BUCKET_NAME = os.getenv('AWS_BUCKET_NAME')
+
+assert(QINIU_ACCESS_KEY and QINIU_SECRET_KEY and QINIU_BUCKET_NAME)
+
+def progress_handler(progress, total):
+    print("{}/{} {:.2f}".format(progress, total, progress/total*100))
+
+# local_file: local file path
+# remote_name: 上传到七牛后保存的文件名
+def upload_to_qiniu(local_file, remote_name, ttl=3600):
+    print('uploading to qiniu', local_file, remote_name, ttl)
+    #构建鉴权对象
+    q = Auth(QINIU_ACCESS_KEY, QINIU_SECRET_KEY)
```

Comment:
We don't have QINIU-ACCESS_KEY and QINIU-SECRET_KEY. We are uploading the issue using Alibaba Cloud